cicecore/cicedynB/general/ice_init.F90

```diff
@@ -1216,8 +1216,8 @@ subroutine input_data
 
       if (trim(atmbndy) == 'default') then
          if (my_task == master_task) then
-            write(nu_diag,*) subname//' WARNING: atmbndy = default'
-            write(nu_diag,*) subname//' WARNING:   For backward compability, set atmbndy = similarity'
+            write(nu_diag,*) subname//' WARNING: "atmbndy = default" is decrecated'
```

Comment:
spelling error: deprecated
Are the double quotes needed in these write statements?

Reply:
Thank you!